I did alot of things (read desc) --removed glamourus--

jmxamongusmodder opened the pull request:
FIXED
Fixed icons (yes you got some icons incorrect)
Gose song added (the secret song)
BF Now Uses Right Sprites (Not Finished)
Improved Shooting (Knockout works rn make this a draft idk how)
Hid Offical Weeks
Bone Notes (SANS BONE NOTES)
WIP MugMan Dying
More Spritesheets to help you with the other weeks (they dont have jsons but yes)

jmxamongusmodder committed:
More Features

New Mad Cuphead Icon
Bone Notes
The Song Of GOSE
Better Cuphead Attacks
Mugman Dying (WIP)

jmxamongusmodder committed:
MORE CHANGES

Knockout now has angry cuphead icon
Hid the offical weeks
Fixed bf sprites + made bf more acord to the current week your on (WIP)
Fixed cupheads icons to be better

jmxamongusmodder committed:
Unused Glamerous song re-added

DaveFan19 commented:
did you even ask for permission to use glamerous? if not, i suggest you get it removed from the pr.

Pico8430 commented:
This pr looks better than the other ones before, but i suggest you to remove Glamerous as were only adding in things that are in the mod. If you do that i will absolutely merge this.

DaveFan19 commented:
> This pr looks better than the other ones before, but i suggest you to remove Glamerous as were only adding in things that are in the mod. If you do that i will absolutely merge this.

--you could just merge this and get rid of glamorous yourself if you want--

DaveFan19 commented:
also btw why would cam boom be needed? is it for some song or sum??

jmxamongusmodder commented:
ALRIGHT GUYS HOLD ON ILL REMOVE GLAMOURUS IM SORRY I JUST HAVENT TOUCHED MY THING IN A VERY LONG TIME

jmxamongusmodder commented:
> This pr looks better than the other ones before, but i suggest you to remove Glamerous as were only adding in things that are in the mod. If you do that i will absolutely merge this.

I WILL OK

jmxamongusmodder committed:
NO

jmxamongusmodder committed:
no again

jmxamongusmodder committed:
no again

jmxamongusmodder committed:
Delete mods/data/glamorous directory

jmxamongusmodder committed:
Delete mods/songs/glamorous directory

jmxamongusmodder committed:
no

jmxamongusmodder committed:
SO MUCH BEING REMOVED

jmxamongusmodder committed:
Delete mtt-text2.png

jmxamongusmodder committed:
Delete gf-act-text.png

jmxamongusmodder committed:
Delete MTT-EX.png

jmxamongusmodder committed:
Delete MTT-EX.xml

jmxamongusmodder committed:
Delete Mettaton_assets.xml

jmxamongusmodder committed:
Delete Mettaton_assets.png

jmxamongusmodder committed:
alright i think im done

jmxamongusmodder committed:
Delete Cam Boom Speed.txt

jmxamongusmodder committed:
Delete Cam Boom Speed.lua

jmxamongusmodder commented:
> This pr looks better than the other ones before, but i suggest you to remove Glamerous as were only adding in things that are in the mod. If you do that i will absolutely merge this.

@Pico8430  i fixed it i removed it

jmxamongusmodder committed:
Delete icon-mtt.png

jmxamongusmodder committed:
Delete icon-mttEX.png

jmxamongusmodder committed:
Delete mtt-text1.png

jmxamongusmodder committed:
a tribute to this mod.

an explination on why i havent worked on it in a while

jmxamongusmodder commented:
> also btw why would cam boom be needed? is it for some song or sum??

it was

DaveFan19 commented:
knockout is just straight up more buggy than before, the green shots dont stop when the function ends, which means they will be there for the entire song, and blue shots dont even work sometimes. i recommend you make this a separate copy of this build and fix the errors.

jmxamongusmodder commented:
> knockout is just straight up more buggy than before, the green shots dont stop when the function ends, which means they will be there for the entire song, and blue shots dont even work sometimes. i recommend you make this a separate copy of this build and fix the errors.

sorry im stupid with LUA im so used to source code
